qa/Vagrantfile

```diff
@@ -17,6 +17,14 @@ Vagrant.configure(2) do |config|
         sh.path = "sys/#{platform.type}/bootstrap.sh"
         sh.privileged = true
       end
+
+      if platform.bootstrap
```

Comment:
see https://github.com/elastic/logstash/pull/5346 for another proposal how to solve this, the idea there is to wrap the logic to select this files in an "extended" version of the platform logic, as we try to have a minimalist Vagrantfile, I think that should be the place.